components/eam/src/chemistry/utils/tracer_data.F90

```diff
@@ -1285,7 +1285,7 @@ subroutine read_next_trcdata(state, flds, file )
              cnt3(flds(f)%coords(ZA_TIMDIM)) = 1
              strt3(flds(f)%coords(ZA_TIMDIM)) = recnos(i)
              !!
-             if (file%linoz_v3.eqv..true..or.file%linoz_v2.eqv..true.) then
+             if (file%linoz_v3 .or. file%linoz_v2) then
```

Comment:
In the original form, even when linoz_v2 is true, this if-block would not be executed.